ember-app/app/mixins/cards/card-move.js

```diff
@@ -11,20 +11,28 @@ var CardMoveMixin = Ember.Mixin.create({
     calculateIssueOrder: function(issue_above, issue_below){
       var issue = this.data.card.get("issue.data");
       if(!issue_above && !issue_below){return issue._data.order; }
-      if(!issue_above){ return this.moveToTop(issue, issue_below); }
-      if(!issue_below){ return this.moveToBottom(issue, issue_above); }
+      if(!issue_above){ return this.moveToTop(issue_below); }
+      if(!issue_below){ return this.moveToBottom(issue_above); }
       return this.move(issue, issue_above, issue_below);
     },
     move: function(issue, issue_above, issue_below){
       var above_order = issue_above._data[this.get("orderKey")];
       var below_order = issue_below._data[this.get("orderKey")];
       return (above_order + below_order) / 2;
     },
-    moveToTop: function(issue, issue_below){
-      return (issue_below._data[this.get("orderKey")]) / 2;
+    moveToTop: function(issue_below){
+      var below_order = issue_below._data[this.get("orderKey")]
+      var order = below_order / this.orderMultiplier;
+      while(order < this.minThreshold){ order *= 10; }
```

Comment:
This loop will be infinite if `order <= 0` - should probably guard against that.